components/anchor/index.zh-CN.md

```diff
@@ -31,4 +31,4 @@ title: Anchor
 | 成员 | 说明 | 类型 | 默认值 |
 | --- | --- | --- | --- |
 | href | 锚点链接 | string |  |
-| title | 文字内容 | string\|ReactNode |  |
+| title | 文字内容 | string |  |
```

Comment:
`title` could be a `ReactNode`.

Reply:
Yes, I have tested and `title` could be `RaectNode`. But `string` is not necessary since it is already included in `ReactNode`.

Reply:
So we should keep `ReactNode` only?

Reply:
Yes, Here is [Docs](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7a5dfdb5f6d9cbb438892a0db8c7d195751a984e/types/react/index.d.ts).